code_style.md

```diff
@@ -162,7 +156,15 @@ ECMAScript
   - Be careful mixing arrow functions and regular functions, eg. if one function in a promise chain is an
     arrow function, they probably all should be.
 - Apart from that, newer ES features should be used whenever the author deems them to be appropriate.
-- Flow annotations are welcome and encouraged.
+- While flow annotations are welcome and encouraged consider to migrate the JS file to TypeScript (see next section).
```

Comment:
Let's be a bit stronger here. We're no longer using Flow or adding new annotations really... Perhaps we can just remove this line?